### PR TITLE
RDoc-1394: add setup examples.

### PR DESCRIPTION
**Pull request by garayx**
https://issues.hibernatingrhinos.com/issue/RDoc-1394

### CODE REVIEW

File: Documentation/4.1/Raven.Documentation.Pages/start/installation/running-in-docker-container.markdown [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+{NOTE: SMB / CIFS mounts}
+
+Linux Docker container running under Windows Docker host with sharing volumes [isn't supported due to CIFS protocol usage](../../start/installation/deployment-considerations#storage-considerations) 
+
+{NOTE /}
+
+## Tags
+
+Beside tags matching the exact builds e.g. `4.0.7-ubuntu.16.04-x64` or `4.0.7-windows-nanoserver` each of the repositories contain the following tags for your convenience:
```
Comment (ppekrol) [inline Documentation/4.1/Raven.Documentation.Pages/start/installation/running-in-docker-container.markdown]: 4.1.4

File: Documentation/4.1/Raven.Documentation.Pages/start/installation/running-in-docker-container.markdown [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+- `4.0-latest` this is an alias to `4.0-ubuntu-latest`
```
Comment (ppekrol) [inline Documentation/4.1/Raven.Documentation.Pages/start/installation/running-in-docker-container.markdown]: 4.1

File: Documentation/4.1/Raven.Documentation.Pages/start/installation/running-in-docker-container.markdown [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+- `4.0-ubuntu-latest` is an alias to the latest RavenDB Ubuntu image
```
Comment (ppekrol) [inline Documentation/4.1/Raven.Documentation.Pages/start/installation/running-in-docker-container.markdown]: 4.1

File: Documentation/4.1/Raven.Documentation.Pages/start/installation/running-in-docker-container.markdown [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+- `4.0-windows-nanoserver-latest` is an alias to the latest RavenDB Windows Nano Server image
```
Comment (ppekrol) [inline Documentation/4.1/Raven.Documentation.Pages/start/installation/running-in-docker-container.markdown]: 4.1

File: Documentation/4.1/Raven.Documentation.Pages/start/installation/running-in-docker-container.markdown [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+A RavenDB Server can run on [Docker](https://www.docker.com/) using our official images available in these channels:
+
+- [ravendb/ravendb](https://hub.docker.com/r/ravendb/ravendb/) containing stable, patch, rc and beta images
+- [ravendb/ravendb-nightly](https://hub.docker.com/r/ravendb/ravendb-nightly/) containing our nightly releases
+
+## Platforms
+
+Server images are published on Docker for the following platforms:
+
+- Ubuntu 16.04
```
[the remark this answers is not in the document]
Comment (garayx) [reply Documentation/4.1/Raven.Documentation.Pages/start/installation/running-in-docker-container.markdown]: done

File: Documentation/4.1/Raven.Documentation.Pages/start/installation/running-in-docker-container.markdown (hunk 1 above)
Comment (garayx) [reply Documentation/4.1/Raven.Documentation.Pages/start/installation/running-in-docker-container.markdown]: done

File: Documentation/4.1/Raven.Documentation.Pages/start/installation/running-in-docker-container.markdown (hunk 2 above)
Comment (garayx) [reply Documentation/4.1/Raven.Documentation.Pages/start/installation/running-in-docker-container.markdown]: done

File: Documentation/4.1/Raven.Documentation.Pages/start/installation/running-in-docker-container.markdown (hunk 3 above)
Comment (garayx) [reply Documentation/4.1/Raven.Documentation.Pages/start/installation/running-in-docker-container.markdown]: done

File: Documentation/4.1/Raven.Documentation.Pages/start/installation/running-in-docker-container.markdown (hunk 4 above)
Comment (garayx) [reply Documentation/4.1/Raven.Documentation.Pages/start/installation/running-in-docker-container.markdown]: done

**Pull request merged**